src/schema/v2/OrderedSet/updateOrderedSetMutation.ts

```diff
@@ -0,0 +1,135 @@
+import {
+  GraphQLString,
+  GraphQLUnionType,
+  GraphQLObjectType,
+  GraphQLBoolean,
+} from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { GraphQLNonNull } from "graphql"
+import { ResolverContext } from "types/graphql"
+import { OrderedSetType } from "./index"
+import {
+  formatGravityError,
+  GravityMutationErrorType,
+} from "lib/gravityErrorHandler"
+
+type OwnerType =
+  | "Artist"
+  | "Artwork"
+  | "Feature Link"
+  | "Gene"
+  | "Ordered Set"
+  | "Partner Show"
+  | "Profile"
+  | "Sale"
+  | "User"
+
+type ItemType = "Fair" | "Feature" | "Sale"
+
+type LayoutType = "default" | "full"
+
+interface Input {
+  description: string
+  id: string
+  itemId: string
+  itemType: ItemType
+  key: string
+  layout: LayoutType
+  name: string
+  ownerType: OwnerType
+  published: boolean
+}
+
+const SuccessType = new GraphQLObjectType<any, ResolverContext>({
+  name: "updateOrderedSetSuccess",
+  isTypeOf: (data) => data.id,
+  fields: () => ({
+    set: {
+      type: OrderedSetType,
+      resolve: (orderedSet) => orderedSet,
+    },
+  }),
+})
+
+const FailureType = new GraphQLObjectType<any, ResolverContext>({
+  name: "updateOrderedSetFailure",
+  isTypeOf: (data) => data._type === "GravityMutationError",
+  fields: () => ({
+    mutationError: {
+      type: GravityMutationErrorType,
+      resolve: (err) => err,
+    },
+  }),
+})
+
+const ResponseOrErrorType = new GraphQLUnionType({
+  name: "updateOrderedSetResponseOrError",
+  types: [SuccessType, FailureType],
+})
+
+export const updateOrderedSetMutation = mutationWithClientMutationId<
+  Input,
+  any | null,
+  ResolverContext
+>({
+  name: "updateOrderedSetMutation",
+  description: "updates an ordered set.",
+  inputFields: {
+    description: { type: GraphQLString },
+    id: { type: new GraphQLNonNull(GraphQLString) },
+    itemId: { type: GraphQLString },
```

Comment:
Would these arguments be used for a PUT to a set? I imagine it's just a few of the other fields like `key`, `published`, etc.

Reply:
Right now, I think the edit sets page will likely be a slightly modified version of the creation, and support updating all these fields.